core/jobs_registry.py

```diff
@@ -52,11 +52,13 @@
     exp_jobs_one_off.ExplorationMigrationJobManager,
     exp_jobs_one_off.ExplorationValidityJobManager,
     exp_jobs_one_off.HintsAuditOneOffJob,
+    exp_jobs_one_off.InteractionAuditOneOffJob,
```

Comment:
Looking at the original job, it says that it's a demonstration job that's not meant for production. Can we add the code for this to the wiki page about "how to write one-off jobs", or write such a wiki page if it doesn't exist -- and then delete the job?